src/core/lib/kbn_internal_native_observable/index.js

```diff
@@ -1,4 +1,4 @@
-import symbolObservable from 'symbol-observable';
+import { observable as Symbol_observable } from 'rxjs/internal/symbol/observable';
```

Comment:
Why change the variable name away from camelCase? We don't generally use variables with underscores in them.

This applies to the other file as well.

Reply:
Ah, true. I got carried away using the same naming as rxjs.